### PR TITLE
Fix npm audit report - add typings (part 2)

### PR DESCRIPTION
**Pull request by emilianox**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -8,14 +8,25 @@
     "removeComments": false,
     "sourceMap": false,
     "target": "ES5",
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "lib": [
```
Comment (matheusml) [inline tsconfig.json]: What is this?

File: src/interfaces/Helpers.ts [hunk 2]
```diff
@@ -1 +1,6 @@
-export type mapToProps = (state: any, ownProps?: any) => any;
+import Store from "./Store";
+
+export type mapToProps = (state: object, ownProps?: object) => object;
+export type ActionsBase<State> = (
```
Comment (matheusml) [inline src/interfaces/Helpers.ts]: Why is this being exported? I think it's not being used

File: tsconfig.json (hunk 1 above)
Comment (emilianox) [reply tsconfig.json]: this is for better es5 support
for example in src/devtools/devtoolsMiddleware.ts:17 find method is added in es6
i only added the "es2015.core" but if not defined ts add automatically 
► For --target ES5: DOM,ES5,ScriptHost

File: src/interfaces/Helpers.ts (hunk 2 above)
Comment (emilianox) [reply src/interfaces/Helpers.ts]: there are utilities you can use in your definitions
I need to add some examples to use it.
if you want, I can remove it for now and later(with examples) re-add it

Comment (matheusml) [reply src/interfaces/Helpers.ts]: Yes, please do that.

Comment (emilianox) [reply src/interfaces/Helpers.ts]: done

File: tsconfig.json (hunk 1 above)
Comment (emilianox) [reply tsconfig.json]: more information in:
https://www.typescriptlang.org/docs/handbook/compiler-options.html#compiler-options
(lib option)

File: src/react/components/connect.tsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ import * as React from "react";
 import shallowEqual from "../../utils/shallowEqual";
 import propValidation from "../../utils/propsValidation";
 import bindActions from "../../utils/bindActions";
-import { mapToProps } from "../../interfaces/Helpers";
+type mapToProps = (state: object, ownProps?: object) => object;
```
Comment (matheusml) [inline src/react/components/connect.tsx]: Why did you change this?

Comment (emilianox) [reply src/react/components/connect.tsx]: This is because you don't want to export the mapToProps typing, I had to duplicate in two files.

**Comment by matheusml** [comment]
💯
